### DISCUSSION TITLE
Function of the Red LED

### DISCUSSION THREAD

**Discussion by PaluZoli** [tbnobody/OpenDTU#381] [General]
Hi,
Thank you all for your development. I'm really impressed with the result.
I have flashed the latest binary to an ESP32-WROOM-32. It recognizes the nRF24L01+. I need to wait till the Sun comes out again to see the data. Until then I'm curious what represents the Red LED on the ESP32. I see no pattern in the flashing. First idea was the communication with the Inverter, but I have set the poll interval to 10s. It is flashing more oft and totally random.
Please advice
Zoltan

**Comment by tbnobody** [comment]
Hi,
the LED is not controlled by any software function. As you can see in this schematic: https://wiki.geekworm.com/images/b/bd/NodeMCU-Schematic-diagram.jpg it just  shows that power is supplied to the board

**Comment by scorpi11** [comment]
Only the blue LED can be used by software running on the ESP32.

I added a blink signal that is used when the radio module is connected and OpenDTU received inverter data during the last minute. As a result, my OpenDTU device blinks with the blue LED whenever data is available from an inverter.

https://github.com/scorpi11/OpenDTU/tree/blink

**Comment by PaluZoli** [comment]
Hi, 
My development board from AZ-Delivery with ESP32-WROOM-32 has only one LED. A red one connected to the ESP32 TXD0 (pin35).
Still I do not understand why does it communicate over the TXD0 pin during normal operation, when it is supplied with a PSU not through an USB of a PC.
Any Idea, is it normal?

**Comment by 2-late-Flo** [comment]
So is it right, that it isn't possible to deactivate the red LED via software? 
I have printed a white housing for the openDTU, but the red LED lights so bright that it is seen through the White housing ☹️

**Reply by scorpi11** [reply]
I think it is hard-wired to the power supply, so there is no way to disable it in software.

**Comment by Solarteur** [comment]
In another case, I toned down the glow of the LED with a dark "permanent marker". The glow can be reduced even further with a small piece of black electrical tape.
